### ISSUE TITLE
Mac Pro 2013 Trashcan 12 Core As a Server - Question Regarding Memory & Graphics Spec Requirements

### ISSUE THREAD

**Issue by Loupap** [apohl79/audiogridder#1347]
### Discussed in https://github.com/apohl79/audiogridder/discussions/1346

<div type='discussions-op-text'>

<sup>Originally posted by **Loupap** March 10, 2024</sup>
Hello Everyone,

Just looking for some guidance here on what Audiogridder basically relies on in order to work without any glitches.

Right now I am using 2 identical fully maxed out spec wise 12 core Macpro 2013 trashcans and the system works great. One is the host and on is the audiogridder server.

I am looking to purchase another Mac Pro 2013 Trashcan 12 core machine that is not fully maxed out in the Memory or Graphics Card Specs. This maschine would be network in to use as another Audiogridder Server in my setup.

Does Audiogridder strictly rely on CPU power (12 Core) from the Server.....or does memory and graphics also have an impact on the effectiveness of the server computers.

In other words if I purchase this 12 core machine...does having 32 gigs or 64 gigs of ram have an impact on the performance on the server computer. Same question regarding the 3 Gig Graphic Cards vs the 6 Gig Graphic cards options on this type of computer.

Thanks in advance.

Louis
</div>
